tools/oesign/oesign.c

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#include <string.h>
```

Comment:
I would avoid modifying the oesign tool right now since we don't intend to make the KSS functionality broadly exposed to users yet, and public changes to the tools generally go through a design review process. 

For the purposes of your investigations, it seems the addition of an internal `OE_SET_ENCLAVE_SGX_KSS` macro is sufficient for your testing needs.